2019-11-28 23:42:51.355 - [Info] MITM Version: 1.26
2019-11-28 23:42:51.361 - [Info] Auto Access Enabled: true
2019-11-28 23:42:51.361 - [Debug] [Init] Auto Access Barrier: true
2019-11-28 23:42:51.384 - [Info] SSH man-in-the-middle server for 172.20.0.6 listening on 0.0.0.0:10105
2019-11-29 00:04:42.705 - [Debug] [Connection] Attacker connected: 49.232.60.2 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-29 00:04:44.174 - [Debug] [Auth] Attacker 49.232.60.2 trying to authenticate with "password"
2019-11-29 00:04:44.175 - [Debug] [Auto Access] Attacker: 49.232.60.2, Threshold: 1, Attempts: 1
2019-11-29 00:04:44.175 - [Debug] [Auto Access] Compromising the honeypot
2019-11-29 00:04:44.175 - [Debug] [Auto Access] Adding the following credentials: "root:Resultat2017"
2019-11-29 00:04:44.320 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-29 00:04:44.320 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.6
2019-11-29 00:04:44.374 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-29 00:04:45.332 - [Debug] [Connection] Attacker connected: 49.232.60.2 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-29 00:04:46.793 - [Debug] [Auth] Attacker 49.232.60.2 trying to authenticate with "password"
2019-11-29 00:04:46.794 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-29 00:04:46.795 - [Debug] [Auth] Attacker: 49.232.60.2 has so far made 1 attempts. Remaining: 5 attempts
2019-11-29 00:04:47.294 - [Debug] [Connection] Attacker closed the connection
2019-11-29 00:04:48.252 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-29 00:15:56.161 - [Debug] [Connection] Client error on ssh server
2019-11-29 00:15:56.162 - [Debug] [Connection] Attacker closed connection
2019-11-29 00:15:56.169 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-29 00:15:56.170 - [Debug] [LXC] Container's OpenSSH server closed connection
